components/Input/Input.tsx

```diff
@@ -0,0 +1,73 @@
+import styles from "./Input.module.css";
+import { ChangeEvent, useState } from "react";
+import classnames from "classnames";
+import eyeOnIcon from "@/assets/images/eye-on.svg";
+import eyeOffIcon from "@/assets/images/eye-off.svg";
+import Image from "next/image";
+
+interface InputProps {
+  isPassword?: boolean;
+  invalidText?: string;
+  onBlur?: () => void;
+}
+
+const Input = ({ isPassword, invalidText, onBlur }: InputProps) => {
+  const [value, setValue] = useState("");
+  const [showPassword, setShowPassword] = useState(false);
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setValue(e.target.value);
+  };
+
+  const handleBlur = () => {
+    if (onBlur) {
```

Comment:
이런식으로 간단하게 써줄수도 있답니다 :)

```suggestion
    onBlur?.();
```